README.md

```diff
@@ -355,6 +355,12 @@ Please read [the following guide](./TROUBLESHOOTING.md) if the plugin is not wor
 
 If you are interested in contributing, please check out the [contribution guidelines](https://github.com/dart-code-checker/dart-code-metrics/blob/master/CONTRIBUTING.md). Feedback and contributions are welcome!
 
+## How to reach us
+
+Please feel free to ask any questions about this tool. Join our community chat on Telegram. We speak English and Russian.
+
+[![Telegram](https://badgen.net/badge/icon/telegram?icon=telegram&label)](https://t.me/DartCodeMetrics)
```

Comment:
It's not clear that the badge is clickable, I'd suggest to make `chat on Telegram` a link and remove the badge.